go.sum

```diff
@@ -1798,6 +1798,8 @@ github.com/stretchr/testify v1.8.3/go.mod h1:sz/lmYIOXD/1dqDmKjjqLyZ2RngseejIcXl
 github.com/stretchr/testify v1.8.4 h1:CcVxjf3Q8PM0mHUKJCdn+eZZtm5yQwehR5yeSVQQcUk=
 github.com/stretchr/testify v1.8.4/go.mod h1:sz/lmYIOXD/1dqDmKjjqLyZ2RngseejIcXlSw2iwfAo=
 github.com/subosito/gotenv v1.2.0/go.mod h1:N0PQaV/YGNqwC0u51sEeR/aUtSLEXKX9iv69rRypqCw=
+github.com/susesgartner/shepherd v0.0.0-20240307220711-9970e2c20892 h1:dpamiqtO1Uk9m704dtztbMMNluRhdqGm1EUzK+ucGc8=
```

Comment:
This will be resolved once you run `go mod tidy` after not pointing to your forked repo.